{
  "id": "CVE-2024-9620",
  "sourceIdentifier": "secalert@redhat.com",
  "published": "2024-10-08T17:15:57.357",
  "lastModified": "2024-10-10T12:56:30.817",
  "vulnStatus": "Awaiting Analysis",
  "cveTags": [

  ],
  "descriptions": [
    {
      "lang": "en",
      "value": "A flaw was found in Event-Driven Automation (EDA) in Ansible Automation Platform (AAP), which lacks encryption of sensitive information. An attacker with network access could exploit this vulnerability by sniffing the plaintext data transmitted between the EDA and AAP. An attacker with system access could exploit this vulnerability by reading the plaintext data stored in EDA and AAP databases."
    },
    {
      "lang": "es",
      "value": "Se encontró una falla en la automatización basada en eventos (EDA) en Ansible Automation Platform (AAP), que carece de cifrado de información confidencial. Un atacante con acceso a la red podría aprovechar esta vulnerabilidad al rastrear los datos de texto sin formato transmitidos entre EDA y AAP. Un atacante con acceso al sistema podría aprovechar esta vulnerabilidad al leer los datos de texto sin formato almacenados en las bases de datos de EDA y AAP."
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "secalert@redhat.com",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
          "baseScore": 5.3,
          "baseSeverity": "MEDIUM",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "NONE",
          "userInteraction": "NONE",
          "scope": "UNCHANGED",
          "confidentialityImpact": "LOW",
          "integrityImpact": "NONE",
          "availabilityImpact": "NONE"
        },
        "exploitabilityScore": 3.9,
        "impactScore": 1.4
      }
    ]
  },
  "weaknesses": [
    {
      "source": "secalert@redhat.com",
      "type": "Secondary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-319"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://access.redhat.com/security/cve/CVE-2024-9620",
      "source": "secalert@redhat.com"
    },
    {
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2317129",
      "source": "secalert@redhat.com"
    }
  ]
}